mote/projects/base/library/browser/patterns/atoms/image/variations/srcset/data.yaml

```diff
@@ -1,7 +1,7 @@
-srcset:
-    -
-        src: foo@200.jpeg
-        dimensions: 200w 100h
-    -
-        src: foo@800.jpeg
-        dimensions: 800w 400h
+#srcset:
```

Comment:
I commented out all this to ensure that dummy data does not leak through to production.